src/components/Todo/Todo.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState, useEffect} from 'react';
+import ItemList from '../ItemList/ItemList';
+import InputItem from '../InputItem/InputItem';
+import Footer from '../Footer/Footer';
+import styles from './Todo.module.css';
+
+const Todo = () => {
+
+	const initialState = {
+		items: [
+			{
+				value: 'Покормить кота',
+				isDone: true,
+				id: 1
+			},
+			{
+				value: 'Сходить на работу',
+				isDone: false,
+				id: 2
+
+			},
+			{
+				value: 'Полить цветы',
+				isDone: true,
+				id: 3
+			}
+		],
+		count: 3,
+	};
+
+const [items, setItems] = useState(initialState.items);
+const [count, setCount] = useState(initialState.count);
+
+const onClickDone = id => {
+	const newItemList = items.map(item => {
+		const newItem = {...item};
+		if (item.id === id) {
+			newItem.isDone = !item.isDone
+		}
+
+		return newItem;
+	});
+
+	setItems(newItemList);
+};
+
+const onClickDelete = id => {
+        const newItemList = items.filter(item => item.id !== id);
+        setItems(newItemList);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)